packages/firebase_vertexai/firebase_vertexai/lib/src/live_api.dart

```diff
@@ -0,0 +1,302 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'api.dart';
+import 'content.dart';
+import 'error.dart';
+
+/// Represents the available voice options for speech synthesis.
+enum Voices {
```

Comment:
If you use a singular enum name the usage will read more naturally. Singular names are more common for Dart enums than plural names. How about naming this `Voice`.

Reply:
changed